src/main/java/seedu/duke/Duke.java

```diff
@@ -25,8 +26,9 @@ public Duke() {
         ui = new Ui();
         storage = new Storage();
         try {
-            taskList = new TaskList(TaskList.deserialize(storage.loadData()));
-            lessonList = new LessonList(LessonList.deserialize(storage.loadData()));
+            taskList = new TaskList(TaskList.deserialize(storage.loadData("tasks.txt")));
+            lessonList = new LessonList(LessonList.deserialize(storage.loadData("lessons.txt")));
```

Comment:
Do you want to make "tasks.txt" and "lessons.txt" global constants instead, or maybe public constants in `Storage.java`? So if they're public this line would look like `taskList = new TaskList(TaskList.deserialize(storage.loadData(Storage.LESSON_FILE_NAME)));`. Same thing for the delete commands.

Reply:
Good point! I will try to make them public constants in `Storage.java`.